activerecord/test/cases/connection_adapters/schema_cache_test.rb

```diff
@@ -28,8 +28,8 @@ def test_yaml_dump_and_load
         cache.connection = @connection
 
         assert_no_queries do
-          assert_equal 12, cache.columns("posts").size
-          assert_equal 12, cache.columns_hash("posts").size
+          assert_equal 13, cache.columns("posts").size
```

Comment:
I agree with Nikita on this -- rather than changing the `Post` model, can we create a new model that has a UUID primary key so that the tests are closer to a real use case? This will also minimize the changeset and make it easier for others to review 😄